pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "python-multipart"
+version = "0.0.5"
+description = "A streaming multipart parser for Python"
+authors = [
+    {name = "Andrew Dunham"},
+]
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {text = "Apache"}
+classifiers = [
+        'Development Status :: 5 - Production/Stable',
+        'Environment :: Web Environment',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python :: 3 :: Only',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
+        'Topic :: Software Development :: Libraries :: Python Modules'
+]
+dependencies = [
+    "atomicwrites==1.2.1",
```

Comment:
Dependencies should not be hard pinned here as it forces ALL users of this lib to use SPECIFICALLY this version. More relaxed requirements as `atomicwrites>=1.0.0` are prefered. The minimum version that works, the better, that allows users more freedom to pin in their apps without a dependency (that's even transitive in the case of fastapi users) forcing you to certain version that might have a security vuln for example